sp800-63-3/cover.md

```diff
@@ -91,14 +91,14 @@ http://dx.doi.org/10.6028/NIST.SP.XXX
 
 {::comment}
 
-**Comments on this publication may be submitted to eauth-comments@nist.gov  
+**Comments on this publication may be submitted to eauth-comment@nist.gov  
```

Comment:
The change was made because we were inconsistent; eauth-comments@ was used on previous revisions of 800-63 and eauth-comment@ was used for the latest NTR. If you want to change it to something more consistent with the current title, we can do that too, but let's make them consistent.